### PR TITLE
Added Missing dependency

### PR DESCRIPTION
**Pull request by Mhmdrza**
**Summary**
I'd followed all along with docs and executed all steps but in the end when I was trying to run <code>gatsby develop</code>
I'd got the following error:

This dependency was not found:
⠀
* netlify-cms-app in ./node_modules/gatsby-plugin-netlify-cms/cms.js
⠀
To install it, you can run: npm install --save netlify-cms-app

### CODE REVIEW

**Comment by talves** [comment]
See [Here][1] for reference to docs for Gatsby that can be applied here. I am wondering if we should include the link at the bottom as a reference.

[1]: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-netlify-cms/README.md

File: website/content/docs/gatsby.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ We can see above that each blog post has a title, a date, a description and a bo
 First let's install some dependencies. We'll need `netlify-cms` and `gatsby-plugin-netlify-cms`. Run the following command in the terminal at the root of your site:
```
[the remark this answers is not in the document]
Comment (Mhmdrza) [reply website/content/docs/gatsby.md]: yea sure

**Comment by Mhmdrza** [comment]
Gatsby.md file seems fine, Is it?

File: website/content/docs/gatsby.md [hunk 2]
```diff
@@ -40,10 +40,10 @@ We can see above that each blog post has a title, a date, a description and a bo
 
 ## Add Netlify CMS to your site
 
-First let's install some dependencies. We'll need `netlify-cms` and `gatsby-plugin-netlify-cms`. Run the following command in the terminal at the root of your site:
+First let's install some dependencies. We'll need `netlify-cms`, `netlify-cms-app` and `gatsby-plugin-netlify-cms`. Run the following command in the terminal at the root of your site:
```
Comment (talves) [inline website/content/docs/gatsby.md]: Change:

```
We'll need `netlify-cms`, `netlify-cms-app` and `gatsby-plugin-netlify-cms`.
```
to
```
We'll need `netlify-cms-app` and `gatsby-plugin-netlify-cms`.
```

File: website/content/docs/gatsby.md [hunk 3]
```diff
@@ -40,10 +40,10 @@ We can see above that each blog post has a title, a date, a description and a bo
...
 
 ```sh
-npm install --save netlify-cms gatsby-plugin-netlify-cms
+npm install --save netlify-cms netlify-cms-app gatsby-plugin-netlify-cms
```
Comment (talves) [inline website/content/docs/gatsby.md]: Change:

```
npm install --save netlify-cms netlify-cms-app gatsby-plugin-netlify-cms
```
to
```
npm install --save netlify-cms-app gatsby-plugin-netlify-cms
```

File: website/content/docs/gatsby.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mhmdrza) [reply website/content/docs/gatsby.md]: I thought that "Also" means that it should be additive

File: website/content/docs/gatsby.md (hunk 3 above)
Comment (Mhmdrza) [reply website/content/docs/gatsby.md]: done;

File: website/content/docs/gatsby.md (hunk 2 above)
Comment (Mhmdrza) [reply website/content/docs/gatsby.md]: done;

**Pull request merged**